datafusion/tests/sql.rs

```diff
@@ -944,7 +944,7 @@ async fn csv_query_nullif_divide_by_0() -> Result<()> {
     let expected = vec![
         vec!["258"],
         vec!["664"],
-        vec!["NULL"],
```

Comment:
I've changed part of the expected Null value representation in this file, from "NULL" to "" to make NULL representation consistent in the file itself.  (some of NULL are represented as "NULL" and others are empty string already).  and follow that in arrow2 at the same time.